vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -74,61 +78,87 @@ func computeETag(data []byte) string {
 	return fmt.Sprintf("\"%X\"", sha512.Sum512(data))
 }
 
+func (c *cache) Get() ([]byte, string, error) {
+	c.once.Do(func() {
+		c.bytes, c.etag, c.err := c.BuildCache()
```

Comment:
I think that'll fail to compile because you can't use the `:=`, but should use `=`. Golang 🤷

Reply:
Thanks for catching that, serves me right for not running `go build` prior to pushing...